jaxws-ejb/pom.xml

```diff
@@ -120,6 +121,15 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
```

Comment:
This would instead be our fork "org.jboss.spec.jakarta.xml.ws:jboss-jakarta-xml-ws-api_4.0_spec", which version is already managed... BUT perhaps it makes sense to instead add this artifact to the jaxws-client BOM (and then you would not need to add it here, i.e. no changes at all for this PR except removing the comments on root and dist POMs)?

Reply:
If you add it to the jaxws-client BOM  I'll test it and withdraw this PR if possible.

Reply:
FYI BOMs PR adding org.jboss.spec.jakarta.xml.ws:jboss-jakarta-xml-ws-api_4.0_spec to the jaxws-client BOM was merged now, so this PR only really needs to remove the comments disabling this QS on root and dist pom.xml